steampipecloud/table_steampipecloud_member.go

```diff
@@ -2,10 +2,8 @@ package steampipecloud
 
```

Comment:
Can you please rename this table + references to `table_steampipecloud_org_member`?

Reply:
updated